example/main.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:ffi';
```

Comment:
License header.

Reply:
Adding a license header will make that header show up on the pub website. For example in package:http it is omitted: https://pub.dev/packages/http#-example-tab-.